packages/editor-ui/src/utils/nodeTypesUtils.ts

```diff
@@ -495,6 +511,19 @@ export const getNodeAuthFields = (nodeType: INodeTypeDescription | null): INodeP
 	return authFields;
 };
 
+export const isNodeFieldMatchingNodeVersion = (
+	nodeField: INodeProperties,
+	nodeVersion: number | undefined,
+) => {
+	if (!nodeVersion) {
+		return true;
+	}
+	if (nodeField.displayOptions?.show && '@version' in nodeField.displayOptions.show) {
```

Comment:
Nit: This could be simplified to just a single if:
```js
if (nodeVersion && nodeField.displayOptions?.show?.['@version']) {
		return nodeField.displayOptions.show['@version']?.includes(nodeVersion);
	}
	return true;
```

Reply:
Alright, agree.